content/docs/reference/reference.json

```diff
@@ -20,81 +20,12 @@
   "autocert": {
     "id": "autocert",
     "path": "/autocert",
-    "title": "Autocert",
-    "description": "Turning on autocert allows Pomerium to automatically retrieve, manage, and renew public facing TLS certificates from Lets Encrypt.",
-    "short_description": "",
+    "title": "Autocert Settings",
+    "description": "Enable Autocert and Autocert settings to retrieve, manage, and renew public-facing TLS certificates from Let's Encrypt.",
+    "short_description": "Autocert manages client-facing TLS certificates from Let's Encrypt.",
     "services": ["authenticate", "proxy"],
     "type": "bool"
   },
-  "autocert-ca": {
```

Comment:
Instead of deleting the individual settings / config options, I think it makes sense to keep the entries and link to the consolidated path (e.g. so you can still search for the individual thing).